src/lib.rs

```diff
@@ -1253,16 +1219,22 @@ impl Build {
             ToolFamily::Msvc { clang_cl } => {
                 if clang_cl {
                     if target.contains("x86_64") {
-                        cmd.args.push("-m64".into());
+                        if self.cuda {
+                            cmd.args.push("-m64".into());
+                        }
+                        cmd.push_cc_arg("-m64".into());
```

Comment:
I'm not really sure what's going on here, how come the flag is pushed twice?

Reply:
I wasn't too sure about if the nvcc compiler passed through the -m32 and -m64 options, so I added it for both. I think it does though, so I'll modify these lines to just add the options onto the command args directly, as both msvc and nvcc support these options.

Also, for context, the nvcc compiler [only supports 32/64 bit cross-compilation](https://stackoverflow.com/a/10582406), and it's phasing out 32 bit compilation altogether as well [it seems](https://devtalk.nvidia.com/default/topic/1026144/cuda-9-0-nvcc-does-not-support-compiling-cuda-code-inside-32-bit-projects-on-visual-studio-2015-/).